src/View/Events/UserEvent.java

```diff
@@ -23,22 +30,106 @@ public UserEvent(AddUserUI addUserUI) {
         this.addUserUI = addUserUI;
     }
 
+    /**
+     * Create a user event reference to table user
+     * @param tableUser where the event is executed
+     */
+    public UserEvent(TableUser tableUser) {
+        this.tableUser = tableUser;
+    }
+
     @Override
     public void actionPerformed(ActionEvent e) {
+        int action = Integer.parseInt(e.getActionCommand());
+        switch(action){
+            case 1://search a user
+                break;
+            case 2://open a form to create a new user
+                updateDialog();
+                break;
+
+            case 3: // accept button to save a user
```

Comment:
what is 3, add constants for these values, they are more represantive and easy to undestand